product_docs/docs/biganimal/release/pricing_and_billing/index.mdx

```diff
@@ -74,8 +76,27 @@ Enabling [Apache Superset](/biganimal/latest/using_cluster/06_analyze_with_super
 
 Enabling [PgBouncer](../getting_started/creating_a_cluster/#pgbouncer) to pool your connections incurs additional costs that depend on your cloud provider. In addition to the cloud provider costs, PgBouncer connects to your primary server and requires an IP address. BigAnimal provisions up to three instances per PgBouncer-enabled cluster to ensure that performance is unaffected, so each availability zone receives its own instance of PgBouncer. The extra VM costs are the 2vcpu SKU times the number of PgBouncer instances. For AWS, the instance type is c5.large. For Azure, the instance type is F2s_v2.
 
-## Billing
+## Payments and billing
+
+On the [Billing](https://portal.biganimal.com/billing) page, you can manage your payment methods and view your invoice total.
+
+Your payment and billing options include:
+
+- Digital self-service using a credit card
+- Direct purchase using the Sales Order Form 
+- Azure Marketplace
+
+### Digital self-service
+You can pay for BigAnimal month-to-month with a credit card by making an [EDB account](https://www.enterprisedb.com/accounts/register). BigAnimal sends you monthly invoices to your email. This invoice includes database costs, and if you're using BigAnimal's cloud, infrastructure costs. 
+
+!!! Note 
+    If you want to pay for a longer term of BigAnimal, like for a year or two, then please contact [Sales](https://info.enterprisedb.com/EDB-Contact-Us.html). 
+
+### Direct purchase
+If you are using BigAnimal's cloud account, you are invoiced on a monthly basis or you can arrange for a longer term.
+
+If you're using your own Microsoft Azure or AWS account, usage details are included in your invoice. Account owners can download a usage report in CSV format from the BigAnimal Usage page and can set the time frame, database type, and cloud provider prior to downloading the report. The information on the page is refreshed hourly.
```

Comment:
This section is a bit confusing. 

> If you're using your own Microsoft Azure or AWS account, usage details are included in your invoice. 

the customers connect their own cloud account (Azure/AWS) to BigAnimal can only download the infrastructure usages that BigAnimal deployed from their own AWS & Azure account. 

They can not download the invoice that EDB charges to them from BigAnimal. 

They can only see the usages report (quantity of each database vcpu-hour) from the usage report. 

So they can not see the cost (in USD) anywhere from BigAnimal if they choose the direct purchase.

Reply:
do you have any response on this @ccestes

Reply:
@drothery-edb Can you take a look at this?

Reply:
@xugy99 Does this clarify things?
> If you're using your own Microsoft Azure or AWS account, database billing is handled by EDB. Details on BigAnimal's 
 Infrastructure  usage from your own cloud account are included in your invoice. Account owners can download a usage report in CSV format from the BigAnimal Usage page. You can set the time frame, database type, and cloud provider prior to downloading the report. The information on the page refreshes hourly.

Reply:
@drothery-edb This looks good.